src/components/MySpace/Profile.tsx

```diff
@@ -135,7 +206,7 @@ function Profile() {
                 </div>
 
                 <div className="profile__body__roles">
-                  <div className="d-flex align-items-center gap-2">
+                  <div style={{ minHeight: "38px" }} className="d-flex align-items-center gap-2">
```

Comment:
move the style to css class

Reply:
yep, have once more suggestion by pratheesh to implement

Reply:
Updated clear button + edit profile button behaviour, 
This minHeight is only in place to prevent the parent box height from increasing when XShare button gets active, I will ask Pratheesh to look into it